lib/server/routes/frames.js

```diff
@@ -349,11 +349,11 @@ FramesRouter.prototype.getFrameById = function(req, res, next) {
  */
 FramesRouter.prototype._definitions = function() {
   return [
-    ['POST', '/frames', this._limiter(defaults), this._verify, this.createFrame],
-    ['PUT', '/frames/:frame', this._limiter(defaults), this._verify, this.addShardToFrame],
-    ['DELETE', '/frames/:frame', this._limiter(defaults), this._verify, this.destroyFrameById],
-    ['GET', '/frames', this._limiter(defaults), this._verify, this.getFrames],
-    ['GET', '/frames/:frame', this._limiter(defaults), this._verify, this.getFrameById]
+    ['POST', '/frames', this.getLimiter(limiter(1000)), this._verify, this.createFrame],
+    ['PUT', '/frames/:frame', this.getLimiter(limiter(1000)), this._verify, this.addShardToFrame],
```

Comment:
Getting this value from config will be useful, so we can adjust as needed. It can be default in https://github.com/Storj/bridge/blob/master/lib/config.js#L58